docker-compose.yml

```diff
@@ -211,6 +218,21 @@ services:
     environment:
       <<: *worker_environment
       WORKER_ADMINISTRATION_REGION: priority
+  worker-graph-schema:
+    image: registry.devshift.net/bayesian/cucos-worker
+    restart: on-failure
+    container_name: coreapi-worker-graph-schema
+    network_mode: bridge
+    links:
+     - gremlin-http
+    command: python3 /graph/create_graph_schema.py
+    environment:
+      BAYESIAN_GREMLIN_HTTP_SERVICE_HOST: "bayesian-gremlin-http"
+      BAYESIAN_GREMLIN_HTTP_SERVICE_PORT: "8182"
+      WORKER_ADMINISTRATION_REGION: ingestion
+      F8A_UNCLOUDED_MODE: 'true'
+    depends_on:
+     - gremlin-http
```

Comment:
Use already existing references (`worker` and `worker_environment`) not to make this duplicit; modify only needed fields (see `worker-ingestion` as an example).